terragrunt/modules/release-distribution/lambdas/doc-router/index.js

```diff
@@ -66,10 +66,17 @@ exports.handler = (event, context, callback) => {
         return temp_redirect('https://www.rust-lang.org/learn', callback);
     }
 
-    // Forward versioned documentation as-is.
-    if (/^\/\d/.test(request.uri)) {
+    // Forward patch versioned documentation as-is.
+    if (/^(\d+)\.(\d+).(\d+)\/(.*)/.test(request.uri)) {
         return callback(null, request);
     }
+    // Include patch version 0 if only minor version is provided
+    // Todo("Forward to latest patch version if possible")
+    const minor_versioned_pattern = /^(\d+)\.(\d+)\/(.*)/
+    if (minor_versioned_pattern.test(request.uri)) {
+        const patched_uri = request.uri.replace(minor_versioned_pattern, "$1.$2.0/$3");
+        return temp_redirect(patched_uri, callback);
```

Comment:
Note: Temp redirect is used so the patch version used can be updated later.

Reply:
why do we want to update it later? 🤔

Reply:
This is in case we could update the code to get the latest patch live and direct to it. 
If we can then temp redirect would be good, as the latest patch version for a minor release can change. 

If we are happy with always redirecting to the patch zero release. Then regular redirect should do.